src/index.ts

```diff
@@ -295,10 +295,24 @@ app.get(
 
 // Status endpoint, which returns a JSON of the last import time per endpoint.
 // This uses our internal event sourcing to determine the last imports.
+// Additionally, it returns a list of alerts if any of the imports are too old.
 app.get('/status', async (req: Request, res: Response, next: NextFunction) => {
     try {
+        const yellowAgeHours = 26
+        const redAgeHours = 30
         const status = await statusApi.getStatus()
-        res.json(status)
+        const alerts = statusApi.getAgeAlerts(
+            status,
+            yellowAgeHours,
+            redAgeHours
+        )
+        if (alerts.yellow || alerts.red) {
+            res.status(500)
```

Comment:
You were probably looking for a way to differentiate between a healthy backend and a non-healthy one, which should cause an error. I wonder if sending a `500` is the right way to go here. At the end of the day, the status request succeeded; it's just that the data is too old.
Curious to hear your thoughts on this, but we can keep it of course to be explicit.

Reply:
it would be ever better to use two http codes but couldn't find any code that would fit

Reply:
btw, we also have the `health` endpoint to check the health. this is the import status endpoint.

Reply:
just for history: @mre doesn't like my pretty if statement ...